OpenRA.Mods.Common/Traits/Demolition.cs

```diff
@@ -91,15 +93,22 @@ public void ResolveOrder(Actor self, Order order)
 					return;
 			}
 
-			self.QueueActivity(order.Queued, new Demolish(self, order.Target, info.EnterBehaviour, info.DetonationDelay,
```

Comment:
Is `order.Queued` save to remove here?

Reply:
Probably not. I didn't notice this overload, reverted